test/lib/y2firewall/clients/auto_test.rb

```diff
@@ -33,11 +33,70 @@
     allow(subject).to receive(:importer).and_return(importer)
   end
 
+  describe "#general_summary" do
+    it "creates a text which includes firewalld overview" do
+      expect(firewalld).to receive(:running?).and_return(true)
+      expect(firewalld).to receive(:enabled?).and_return(true)
```

Comment:
These calls shouldn't be expectations at all.

Reply:
why? the summary builds on to of results of these calls.

Reply:
of course but it is not what u are testing, or is it?